src/sql-lexer-wasm/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "mz-sql-lexer-wasm"
+description = "The lexer for Materialize's SQL dialect, with wasm build targets."
+version = "0.0.0"
+edition = "2021"
```

Comment:
I don't love having to hard code edition and rust-version here because there's some build script that errors otherwise. Hmm maybe easier to fix the build script.

Reply:
https://buildkite.com/materialize/tests/builds/63694#018a8b66-a7d4-4c80-a8ce-d77cecaa38c3 is that error

Reply:
I think since we explicitly exclude this crate in the the workspace's Cargo.toml we cannot inherit it. There doesn't appear to be a way to force that behavior either. Testing things locally, it seems like we can drop rust-version and have it build just fine. That changes far more frequently than edition, so perhaps that's okay?